web/src/app/login/LoginComponent.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class LoginComponent extends Component {
+  render() {
+    const link = 'api/login';
+    return (
+      <div>
+        <a href={link}>LOGIN WITH TWITCH</a>
```

Comment:
Nothing to fix here, just wanted to call out that internationalization is a huge deal on a lot of projects. Im not sure the best way to handle it, but i think its typically done by putting placeholder text everywhere. then having a jsopn file for every language you want to support.

english.json
login.text = LOGIN WITH TWITCH

spanish.json
login.text = "some spanish words..."

Reply:
Yep, thats exactly what we do for SPSS. We can maybe go back and do that at a later date if we want to internationalize.